src/Sql/dbo/Tables/Cipher.sql

```diff
@@ -10,6 +10,7 @@
     [CreationDate]   DATETIME2 (7)    NOT NULL,
     [RevisionDate]   DATETIME2 (7)    NOT NULL,
     [DeletedDate]    DATETIME2 (7)    NULL,
+    [Reprompt]       TINYINT          NOT NULL,
```

Comment:
A bit of storage space on the cipher table to be adding another `NOT NULL` column, especially in production as this operation could take a while to run, potentially locking the table for a bit. On this, I would rather see nullable constructs used which can then be sent back to the API caller as not-null, using the default `0` (None) value, but leaving it nullable in the DB (where `null` is unset/default)